spec/models/shelter_reviews_spec.rb

```diff
@@ -0,0 +1,10 @@
+require "rails_helper"
+
+RSpec.describe ShelterReview do
+  describe "relationships" do
+    it {should belong_to :shelter}
```

Comment:
Some spacing improvements, for consistency, but we can do all of that at the end.

Reply:
YES I love that we are on the same page in terms of LOVING good formatting and also trying to prioritize what is important at this moment